### DISCUSSION TITLE
Which deviceSource has GTR 3 Pro?

### DISCUSSION THREAD

**Discussion by melianmiko** [zepp-health/zeppos-forum#233] [Q&A]
Ran intro interesting problem.

All my apps are using `deviceSource` to identify which device user has. And all my apps has support of GTR 3 Pro, which I identify with following sources:

![image](https://github.com/zepp-health/zeppos-forum/assets/67851274/5f885cce-3724-437e-9e7d-711d4772696f)

In simulator, all works fine. But, today I've got an email for user, which GTR 3 Pro didn't have one of those `deviceSource`'s, so due to that all my apps on their watch look like this:

![image](https://github.com/zepp-health/zeppos-forum/assets/67851274/1c64538f-8b51-4f42-88a2-b600515f73a3)
> App: AI Chat (available in Zepp app store)

**So, may GTR 3 Pro have other `deviceSource`, that isn't listed in documentation?**

**Comment by htoooth** [comment]
hi @melianmiko
this device is not GTR3 Pro. You should use 242 of device source if you want to adjust this device.
